discovery/db_discovery.go

```diff
@@ -0,0 +1,100 @@
+package discovery
+
+import (
+	"time"
+
+	"github.com/livepeer/go-livepeer/common"
+	"github.com/livepeer/go-livepeer/core"
+	lpTypes "github.com/livepeer/go-livepeer/eth/types"
+	"github.com/livepeer/go-livepeer/net"
+
+	"github.com/golang/glog"
+)
+
+var cacheRefreshInterval = 1 * time.Hour
+var getTicker = func() *time.Ticker {
+	return time.NewTicker(cacheRefreshInterval)
+}
+
+type DBOrchestratorPoolCache struct {
+	node *core.LivepeerNode
+}
+
+func NewDBOrchestratorPoolCache(node *core.LivepeerNode) *DBOrchestratorPoolCache {
+	if node.Eth == nil {
+		glog.Error("Could not refresh DB list of orchestrators: LivepeerNode nil")
+		return nil
+	}
+
+	ticker := getTicker()
+	go func(node *core.LivepeerNode) {
+		for {
```

Comment:
I believe an alternative to nesting the single case select statement within this for loop is to just use a for loop and range over the `ticket.C` channel:

```
for _ := range ticker.C {
    ...
}
```